sdk/eventgrid/azure-eventgrid/README.md

```diff
@@ -206,10 +209,12 @@ This example publishes a Cloud event.
 import os
```

Comment:
just for checking purposes, have you copy-pasted these into samples and run as is?